Packs/OracleCloudInfrastructure/Integrations/OracleCloudInfrastructureEventCollector/OracleCloudInfrastructureEventCollector_test.py

```diff
@@ -633,3 +633,53 @@ def test_fetch_events(self, mocker, dummy_client, event_list, expected_list, exp
         mocked_handle_fetched_events = mocker.patch("OracleCloudInfrastructureEventCollector.handle_fetched_events")
         main()
         mocked_handle_fetched_events.assert_called_once_with(expected_list, expected_time)
+
+
+class BugDummy:
```

Comment:
No need to add this. You only need to add an example that your new test fails on the error before the code changes and succeeds after it.